COCO_DATA_FORMAT.md

```diff
@@ -94,3 +94,18 @@ Specifically, **only** image files are supported for the label files. The ground
     ]
 }
 ```
+
+## Regression
+
+Here is one example of the json file for regression task, where the "target" in the "annotations" field is a real-valued number (e.g. a score, an age, etc.). Note that each image should only have one regression target (i.e. there should be exactly one annotation for each image).
```

Comment:
In the foreseeable future, we don't have a plan to use a dataset with multiple targets per image?

Reply:
+1

Reply:
@shonohs @cy-bai we can use multitask dataset to organize the multiple target data, where multiple coco will be provided.

Reply:
Sounds good; we can use the multitask dataset format with multiple COCO files if there are multiple targets.